conda_smithy/feedstock_io.py

```diff
@@ -75,7 +75,12 @@ def remove_file(filename):
 
 
 def copy_file(src, dst):
-    shutil.copy2(src, dst)
+    with io.open(src, "r", encoding="utf-8") as fh_src:
+        with io.open(dst, "w", encoding="utf-8", newline="\n") as fh_dst:
+            for line in fh_src:
+                fh_dst.write(line)
+
+    shutil.copymode(src, dst)
```

Comment:
So I had added this to ensure all files that we copy from `conda-smithy` have the right line endings. However, I expect we probably don't need this.